tv_apps/smart-system/js/preview_window.js

```diff
@@ -132,9 +133,28 @@
     localStorage.setItem(PREVIEW_OPENED_TIMES_KEY,
       JSON.stringify(previewOpenedTimes));
 
-    this.systemBanner.show({
-      id: 'preview-app-hint'
-    });
+    var showPreviewHint = function() {
+      window.interactiveNotifications.showNotification(
+        window.InteractiveNotifications.TYPE.NORMAL, {
+          title: {
+            id: 'preview-app-hint'
+          },
+          text: {
+            id: 'add-to-apps'
+          },
+          icon: ADD_TO_APPS_ICON_PATH
+        });
+    };
+
+    if (this.isWebsite) {
+      BookmarkManager.get(this.identity).then((bookmark) => {
+        if (!bookmark) {
+          showPreviewHint();
+        }
+      });
+    } else if (!applications.getByManifestURL(this.manifestURL)) {
```

Comment:
Since an app which has already been added to Apps won't be opened by PreviewWindow, this line seems useless.

BTW, you should use `AppInstallManager.getAppAddedState` if you're really want to check this state here.